rscripts/GWA_TBX/05_WCR_OPT_WaterDetection.R

```diff
@@ -28,24 +28,26 @@
 ##Minimum_mapping_unit = number 3
 ##Plot_water_probability= Boolean False
 ##Plot_certainty_indicator= Boolean False
+##Tile_size= number 2000 
 
 
 # -> Test parameters for execution in R
-DEBUG <- T
+DEBUG <- F
 start_time <- proc.time()
 
 if (DEBUG) {
-  .libPaths("C:\\Users\\ludwig\\.qgis2\\processing\\rlibs")
-  #.libPaths("C:\\Users\\ludwig\\Documents\\R\\win-library\\3.2")
-  Directory_containing_indices= "I:\\WI\\02_InterimProducts\\WI_test\\step3_indices"
-  Directory_containing_TWI= "I:\\WI\\02_InterimProducts\\example_site\\step05_TWI"
-  Output_Directory="I:\\WI\\02_InterimProducts\\WI_test"
+  .libPaths("C:\\Users\\dulleck\\.qgis2\\processing\\rlibs")
+  #.libPaths("C:\\Users\\ludwig\\Documents\\R\\win-library\\3.2")
+  Directory_containing_indices= "T:\\Processing\\2687_GW_A\\03_Products\\GWA-TOOLBOX\\trainingkits_UW2_201802\\WCR\\02_Results\\example_site\\step3_indices"

```

Comment:
@bjoern234 It would be nice if you could remove the hard-coded paths from the source code. Maybe you could keep that DEBUG code block in a text file on your computer and paste it into the file while you prototype and remove it when you commit?